packages/hash/api/src/graphql/resolvers/knowledge/page/update-page.ts

```diff
@@ -0,0 +1,40 @@
+import { WORKSPACE_TYPES } from "../../../../graph/workspace-types";
+import { PageModel } from "../../../../model";
+import {
+  MutationUpdateKnowledgePageArgs,
+  ResolverFn,
+} from "../../../apiTypes.gen";
+import { LoggedInGraphQLContext } from "../../../context";
+import {
+  UnresolvedKnowledgePageGQL,
+  mapPageModelToGQL,
+} from "../model-mapping";
+
+export const updateKnowledgePage: ResolverFn<
+  Promise<UnresolvedKnowledgePageGQL>,
+  {},
+  LoggedInGraphQLContext,
+  MutationUpdateKnowledgePageArgs
+> = async (
+  _,
+  { entityId, updatedProperties },
+  { dataSources: { graphApi } },
+) => {
+  const page = await PageModel.getPageById(graphApi, { entityId });
+
+  const updatedPageEntity = await page.updateProperties(graphApi, {
```

Comment:
Same comment about `*Model`

Reply:
Addressed in https://github.com/hashintel/hash/pull/1157/commits/dc6134dfe601d43f8b534b0b38019276adf29aea